opencti-platform/opencti-front/src/private/components/data/ToolBar.js

```diff
@@ -506,6 +508,17 @@ class ToolBar extends Component {
     });
   }
 
+  titleCopy() {
+    const { t } = this.props;
+    if (this.props.numberOfSelectedElements > maxNumberOfObservablesToCopy) {
+      return t('Copy disabled: too many selected elements');
+    }
+    if (!this.props.handleCopy) {
```

Comment:
Not needed if you hide the button completely. But for the future : not having the handleCopy should be the first condition you check as it is the one with the most 'functional' meaning. In your code, if you select too many elements and don't have handlecopy you will see "Too many selected elements" and not "Copy is only available for observables"